ci/test/00_setup_env_native_fuzz_with_msan.sh

```diff
@@ -17,8 +17,12 @@ export PACKAGES="ninja-build"
 # BDB generates false-positives and will be removed in future
 export DEP_OPTS="DEBUG=1 NO_BDB=1 NO_QT=1 CC=clang CXX=clang++ CFLAGS='${MSAN_FLAGS}' CXXFLAGS='${MSAN_AND_LIBCXX_FLAGS}'"
 export GOAL="install"
+# Setting CMAKE_{C,CXX}_FLAGS_DEBUG flags to an empty string ensures that the flags set in MSAN_FLAGS remain unaltered.
 # _FORTIFY_SOURCE is not compatible with MSAN.
 export BITCOIN_CONFIG="\
+ -DCMAKE_BUILD_TYPE="Debug" \
```

Comment:
If setting the build type specific flags to empty is all that's needed, why are you also changing the build type to Debug, rather than leaving it as RelwithDebInfo (same as oss-fuzz).

Reply:
To pass `linux_debug_CPPFLAGS` from depends.

(I was going to add this explanation to the PR description after more tests completed).